docs/modules/ROOT/pages/tutorials/migration.adoc

```diff
@@ -0,0 +1,619 @@
++
+IMPORTANT: This could be used for jobs, but in the case of independent application parts this should be prevented.
++
+To create a affinity rule for all the pods at once, you can use the https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/[app.kubernetes.io/component label].
++
+[source,yaml]
+--
+affinity:
+  podAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+    - labelSelector:
+        matchExpressions:
+        - key: app.kubernetes.io/component
+           operator: In
+          values:
+          - backend
+      topologyKey: kubernetes.io/hostname
+--
+
+. Perform the migration itself. There are three major mechanisms for this:
+.. Migration with `rsync`
+.. Migration through jobs
+.. Using continuous sync
+
+The following sections provide information about each strategy.
+
+### Migration with `rsync`
+
+The first option to migrate your storage from APPUiO Public to APPUiO Cloud consists in using `rsync`. The manifests below create the required objects.
+
+[source,yaml]
+--
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: rsync-test
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rsync-destination
+  namespace: rsync-test
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  namespace: rsync-test
+  name: rsync-destination
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: edit
+subjects:
+- kind: ServiceAccount
+  name: rsync-destination
+  namespace: rsync-test
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: rsync-destination
+  namespace: rsync-test
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  volumeMode: Filesystem
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: rsync-destination
+  name: rsync-destination
+  namespace: rsync-test
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: rsync-destination
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: rsync-destination
+    spec:
+      containers:
+      - image: registry.access.redhat.com/rhel7/rhel-tools
+        imagePullPolicy: IfNotPresent
+        name: rhel-tools
+        command:
+          - tail
+          - -f
+          - /dev/null
+        volumeMounts:
+        - mountPath: /rsync-destination
+          name: rsync-destination
+      volumes:
+      - name: rsync-destination
+        persistentVolumeClaim:
+          claimName: rsync-destination
+--
+
+=== Job-Based Migration
+
+The second option for migrating your storage from APPUiO Public to APPUiO Cloud consists in using jobs. The manifest below defines the objects required for this strategy.
```

Comment:
```suggestion
The second option for migrating your storage from APPUiO Public to APPUiO Cloud consists in using jobs.
The manifest below defines the objects required for this strategy.
```

Reply:
Again, what's the purpose of these changes?